ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -446,8 +445,8 @@ Loader {
                 showHeader: root.shouldRepeatHeader || dateGroupLabel.visible || isAReply ||
                             (root.prevMessageContentType !== Constants.messageContentType.systemMessagePrivateGroupType && root.senderId !== root.prevMessageSenderId)
                 isActiveMessage: d.isMessageActive
-                topPadding: showHeader ? Style.current.halfPadding : 2
-                bottomPadding: showHeader && nextMessageHasHeader() ? Style.current.halfPadding : 2
+                topPadding: showHeader ? Style.current.halfPadding : 0
```

Comment:
Shouldn't it be 4 the minimum padding between two consecutive messages (2 top + 2 bottom)? 

<img width="1150" alt="Screenshot 2023-02-16 at 11 07 03" src="https://user-images.githubusercontent.com/47811206/219319173-a3945c1f-cdf3-4bc4-a913-37b07daee99d.png">

Reply:
Not when repeating header, ie messages coming from the same author

Reply:
Sorry, I should have added a larger screen shot. It's from messages coming from the same author

<img width="1150" alt="Screenshot 2023-02-16 at 13 02 19" src="https://user-images.githubusercontent.com/47811206/219347879-ae613893-39f5-46c0-970f-5534545f5d04.png">